packages/fxa-settings/src/components/GetDataTrio/index.tsx

```diff
@@ -54,6 +58,35 @@ export const GetDataCopySingleton = ({ value, onAction }: GetDataTrioProps) => {
   );
 };
 
+export const GetDataCopySingletonInline = ({
+  value,
+  onAction,
+}: GetDataTrioProps) => {
+  return (
+    <Localized
+      id="get-data-trio-copy-2"
+      attrs={{ title: true, ariaLabel: true }}
+    >
+      <button
+        title="Copy"
+        type="button"
+        onClick={async () => {
+          const copyValue = Array.isArray(value) ? value.join('\r\n') : value;
+          await copy(copyValue);
+          onAction?.('copy');
+        }}
```

Comment:
Me thinks this is a copy-pasta of L42-46 above.
Not sure if it warrants extracting out in a separate function to keep it slightly DRYer.
Or, if we're getting rid of some copy UI above eventually, feel free to laugh and ignore this comment.

Reply:
Not yet certain if we'll be keeping one of the two or both, so I think I'll keep these separate for now.